workers/am43.py

```diff
@@ -177,8 +177,8 @@ def single_device_status_update(self, device_name, data):
 
         _LOGGER.debug("Updating %s device '%s' (%s)", repr(self), device_name, data["mac"])
 
-        shade = Zemismart.Zemismart(
-            data["mac"], data["pin"], max_connect_time=self.per_device_timeout, withMutex=True)
+        shade = Zemismart.Zemismart(data["mac"], data["pin"], max_connect_time=self.per_device_timeout,
+                                    withMutex=True, iface=data.get('iface'))
```

Comment:
Please add it as example to config.yaml.example. Also I think it should support case when iface is empty ( because someone didn't update config )

Reply:
`None` is a valid `iface` value, so it should work just fine for those who will not update the config.